### PR TITLE
ZCS-8441:  upgrading yui-compressor version to 2.4.8

### PR DESCRIPTION
**Pull request by auminpatel**

### CODE REVIEW

File: ivy.xml [hunk 1]
```diff
@@ -13,13 +13,12 @@
...
   <dependency org="org.apache.httpcomponents" name="httpmime" rev="${httpclient.version}"/>
   <dependency org="com.google.guava" name="guava" rev="23.0" />
-  <dependency org="com.yahoo.platform.yui" name="yuicompressor" rev="2.4.2-zimbra" />
+  <dependency org="com.yahoo.platform.yui" name="yuicompressor" rev="2.4.8" />
   <dependency org="commons-fileupload" name="commons-fileupload" rev="1.2.2" />
   <dependency org="javax.mail" name="mail" rev="1.4.5" />
   <dependency org="net.spy" name="spymemcached" rev="2.12.1" transitive="false"/>
   <dependency org="org.eclipse.jetty" name="jetty-io" rev="9.3.5.v20151012" />
   <dependency org="org.json" name="json" rev="20090211" />
-  <dependency org="dom4j" name="dom4j" rev="1.5.2" />
```
Comment (BhavinBathani) [inline ivy.xml]: why removed dom4j?

**Comment by BhavinBathani** [comment]
Do we need to worry about failing build?

File: ivy.xml (hunk 1 above)
Comment (auminpatel) [reply ivy.xml]: Have reverted the chnage , it has been upgraded by api team

**Comment by auminpatel** [comment]
I dont  think we need to woory about the build failing a we dont have any config.yml file

**Comment by BhavinBathani** [comment]
Please resolve conflict.

**Comment by madhavdhuppe** [comment]
please squash changes

**Comment by auminpatel** [comment]
Closing as some merge conlicts

**Pull request closed without merging**